File: 2894-divisible-and-non-divisible-sums-difference/2894-divisible-and-non-divisible-sums-difference.ts
function differenceOfSums(n: number, m: number): number {
   let divTotal=0;
   let noDivTotal=0;
   for(let i=0;i<=n;i++){
    if(i%m==0){
       divTotal+=i 
    }else{
        noDivTotal+=i
    }
   }
   return noDivTotal-divTotal;
};

File: 2894-divisible-and-non-divisible-sums-difference/2894-divisible-and-non-divisible-sums-difference.cs
public class Solution {
    public int DifferenceOfSums(int n, int m) {
        int divTotal=0;
        int noDivTotal=0;
        for(int i=0;i<=n;i++){
            if(i%m==0){
                divTotal+=i;
            }else{
                noDivTotal+=i;
            }
        }
        return noDivTotal-divTotal;
    }
}